website/docs/api/cli.md

```diff
@@ -455,7 +455,7 @@ improvement.
 
 ```bash
 $ python -m spacy pretrain [texts_loc] [vectors_model] [output_dir]
-[--width] [--depth] [--cnn-window] [--cnn-pieces] [--use-chars] [--sa-depth]
+[--width] [--conv-depth] [--cnn-window] [--cnn-pieces] [--use-chars] [--sa-depth]
 [--embed-rows] [--loss_func] [--dropout] [--batch-size] [--max-length]
```

Comment:
This is outdated (with the configs), we'll need to thoroughly revise the CLI documentation for v.3.

Reply:
Yeah, I've pretty much ignored the docs for now, especially the CLI. I also have a PR up tomorrow that refactors the CLI (using @tiangolo's `typer`). This mostly affects the internals, but it might also open up some new possibilities for CLI arguments.